src/cleanvision/imagelab.py

```diff
@@ -320,9 +515,14 @@ def get_stats(self) -> Any:
         return self.info["statistics"]
 
     def save(self, path: str) -> None:
```

Comment:
overwriting an existing folder could be dangerous if somebody misspecifies the path.
What do you think about adding another argument:
`allow_overwrite = False` by default. 
If it's false, then this method raises ValueError if the path already exists telling user to set this argument to True.

What do you think? @elisno @sanjanag

Reply:
It's a good argument to have. I've added.

Reply:
If we're going to use an `allow_overwrite`, then we need to raise a [FileExistsError](https://docs.python.org/3/library/exceptions.html#FileExistsError).